packages/flutter_tools/templates/package_ffi/build.dart.tmpl

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Is this code copied from the Dart repo? Otherwise this should have the flutter/flutter header, I think.